### ISSUE TITLE
Missing 'mhcflurry_affinity_percentile' values in v2.0.2 output

### ISSUE THREAD

**Issue by abyssum** [openvax/mhcflurry#195]
Hello,

and thank you for this fine piece of software!
I've been using MHCflurry as part of a pipeline regarding neoantigen prediction, in my use-case MHCflurry is installed in a Docker container with the following command (which should install the latest stable version):

`RUN pip install mhcflurry`

I noticed that on your GitHub repo the latest release is v2.0.1, while in the Docker image v2.0.2 is installed via pip. The issue is that for some specific HLA types MHCFlurry v2.0.2 will return an IC50 affinity prediction, but not the affinity percentile (Rank). When I roll back to v2.0.1, MCFlurry will produce the expected results.

I am attaching my run commands and part of the output from the two versions for you to review.

```
$ mhcflurry-predict --version
mhcflurry 2.0.1

$ mhcflurry-predict --affinity-only --alleles HLA-C*01:02  HLA-A*03:01 --peptides SAQGQFSAV AQGQFSAVP QGQFSAVPC GQFSAVPCV QFSAVPCVS FSAVPCVSG SAVPCVSGG AVPCVSGGL --out ./test_201.tsv --models /path/.local/share/mhcflurry/4/2.0.0/models_class1_pan/models.combined --output-delimiter '\t'

$ cat test_201.tsv 
allele	peptide	mhcflurry_affinity	mhcflurry_affinity_percentile
HLA-C*01:02	SAQGQFSAV	60.07806157934322	0.20062500000000005
HLA-C*01:02	AQGQFSAVP	25609.89973204682	34.68475
HLA-C*01:02	QGQFSAVPC	20452.469230754792	20.467000000000013
HLA-C*01:02	GQFSAVPCV	7832.203067779404	7.567875000000004
HLA-C*01:02	QFSAVPCVS	21646.555206274086	22.71350000000001
HLA-C*01:02	FSAVPCVSG	12597.12725724526	10.986000000000002
HLA-C*01:02	SAVPCVSGG	11551.733180526306	10.162500000000003
HLA-C*01:02	AVPCVSGGL	74.8828944368739	0.304125
HLA-A*03:01	SAQGQFSAV	24944.696803879757	11.877999999999995
HLA-A*03:01	AQGQFSAVP	31296.007229554132	31.022125000000006
HLA-A*03:01	QGQFSAVPC	32270.45521977301	41.914375
```
While:

```
$ mhcflurry-predict --version
mhcflurry 2.0.2

$ mhcflurry-predict --affinity-only --alleles HLA-C*01:02  HLA-A*03:01 --peptides SAQGQFSAV AQGQFSAVP QGQFSAVPC GQFSAVPCV QFSAVPCVS FSAVPCVSG SAVPCVSGG AVPCVSGGL --out ./test_202.tsv --models /path/.local/share/mhcflurry/4/2.0.0/models_class1_pan/models.combined --output-delimiter '\t'

$ cat test_202.tsv
allele	peptide	mhcflurry_affinity	mhcflurry_affinity_percentile
HLA-C*01:02	SAQGQFSAV	60.07806157934322	
HLA-C*01:02	AQGQFSAVP	25609.89973204682	
HLA-C*01:02	QGQFSAVPC	20452.469230754792	
HLA-C*01:02	GQFSAVPCV	7832.203067779404	
HLA-C*01:02	QFSAVPCVS	21646.555206274086	
HLA-C*01:02	FSAVPCVSG	12597.12725724526	
HLA-C*01:02	SAVPCVSGG	11551.733180526306	
HLA-C*01:02	AVPCVSGGL	74.8828944368739	
HLA-A*03:01	SAQGQFSAV	24944.696803879757	11.877999999999995
HLA-A*03:01	AQGQFSAVP	31296.007229554132	31.022125000000006
HLA-A*03:01	QGQFSAVPC	32270.45521977301	41.914375
```

I don't know what exactly causes this behavior, but I figured that you should be aware of this issue.

**Comment by timodonnell** [comment]
Good catch, thanks for reporting this. Looks like an issue caused by a change in how we canonicalize allele names. Should be a straightforward fix.

**Comment by timodonnell** [comment]
This should be fixed in mhcflurry 2.0.3
